ui/src/app/components/admin/audit-page-component.tsx

```diff
@@ -1,10 +1,10 @@
 import * as React from 'react';
+import { Navigate } from 'react-router-dom-v5-compat';
 import * as fp from 'lodash/fp';
 
 import { AuditAction, AuditLogEntry } from 'generated/fetch';
 
 import { AuditActionCardListView } from 'app/components/admin/audit-card-list-view';
-import { Navigate } from 'app/components/app-router';
```

Comment:
note that this is not the same component.  It's a custom wrapper that we wrote.

However: it was only used here, and I tested that the functionality didn't change by using the (new) native React Router component